From 7fe6807e3ad4e13107e2d5b2e82db22a80148a2f Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Thu, 21 Nov 2019 15:34:49 +0100
Subject: [PATCH] topic-miami.dtsi: Fix ltc2990 settings

The LTC2990 driver supports all chip modes and needs to be configured
using a lltc,meas-mode setting.
---
 arch/arm/boot/dts/topic-miami.dtsi | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/topic-miami.dtsi b/arch/arm/boot/dts/topic-miami.dtsi
index 72e0fdfc4630..f1702997b0dd 100644
--- a/arch/arm/boot/dts/topic-miami.dtsi
+++ b/arch/arm/boot/dts/topic-miami.dtsi
@@ -116,9 +116,10 @@
 		#gpio-cells = <2>;
 	};
 	/* Current monitoring chip */
-	monitorcurrent: ltc2990@4C {
-		compatible = "ltc2990";
-		reg = <0x4C>;
+	monitorcurrent: ltc2990@4c {
+		compatible = "lltc,ltc2990";
+		reg = <0x4c>;
+		lltc,meas-mode = <6 3>; /* Current sensor mode */
 	};
 	/* EEPROM */
 	eeprom: eeprom@50 {
-- 
2.17.1

